tests/unit/mainwindow/test_prompt.py

```diff
@@ -130,3 +131,9 @@ def test_root_path(self, get_prompt):
         """With / as path, show root contents."""
         prompt = get_prompt('/')
         assert prompt._file_model.rootPath() == '/'
+
+    def test_tab_root(self, get_prompt, qtbot):
+        prompt = get_prompt('/')
+        qtbot.keySequence(prompt._lineedit, QKeySequence(Qt.CTRL + Qt.Key_U))
```

Comment:
[qtbot.keyPress](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot.keyPress) has a `modifiers` argument, so I think you could use:

```suggestion
        qtbot.keyPress(prompt._lineedit, Qt.Key_U, Qt.ControlModifier)
```

(also using the longer name for consistency, the shorter modifier names aren't used in qutebrowser's codebase so far).